src/components/forms/language-panel/language-panel.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Paper, TextField, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { useStyles } from './language-panel.styles';
+import { capitalizeFirstLetter } from '../../../utils';
+
+const LanguagePanel = ({ lang, inputOptions }) => {
+  const styles = useStyles();
+
+  const { values, touched, errors, inputs, handleChange } = inputOptions;
+
+  return (
+    <div className={styles.languagePanel}>
+      <Typography className={styles.title} component='h1' variant='h5'>
+        {lang.toUpperCase()}
+      </Typography>
+      <Paper className={styles.inputPanel}>
+        {inputs.map((input) => {
+          const inputName = lang + capitalizeFirstLetter(input.name);
+
+          return (
+            <>
+              <TextField
+                data-cy={`${lang}-${input.name}`}
+                id={`${inputName}`}
+                className={styles.textField}
+                variant='outlined'
+                label={input.label}
+                error={touched[`${inputName}`] && !!errors[`${inputName}`]}
```

Comment:
You can actually make it simpler just like this `touched[inputName]`

Reply:
You have a couple of places with this, can you please track all of them and change them?

Reply:
fixed